pages/index.tsx

```diff
@@ -16,6 +16,10 @@ const Home: NextPage = () => (
         >
           <h2>install our discord bot</h2>
         </a>
+      <div className={styles.extra-links}>
+          <a href="/help">▶ Starky: wtf?</a>
```

Comment:
Do not use an `<a>` element to navigate to `/help/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages